src/script/start.js

```diff
@@ -21,10 +22,25 @@ const start = async (fileName) => {
 
   // Override webpack config
   // entry
-  targetProjectWebpack.entry = { bundle: path.resolve(__dirname, '..', '..', 'dist', 'gui.build.js') }
+  targetProjectWebpack.entry = {
+    bundle: [
+      'webpack-hot-middleware/client',
+      path.resolve(__dirname, '..', '..', 'dist', 'gui.build.js'),
+    ],
+  }
+  // plugins
+  targetProjectWebpack.plugins = [
+    new webpack.HotModuleReplacementPlugin(),
+    new webpack.NoEmitOnErrorsPlugin(),
+    new webpack.WatchIgnorePlugin([path.resolve(__dirname, '..', '..', 'dist', 'component.js')]),
+  ]
+  // modules resolver fallback
+  targetProjectWebpack.resolve = Object.assign({}, targetProjectWebpack.resolve, {
+    modules: ['node_modules', path.resolve(__dirname, '..', '..', 'node_modules')],
+  })
   // output
   targetProjectWebpack.output = {
-    path: path.resolve('..', '..', 'public'),
+    path: '/',
```

Comment:
Because we are using `webpack-dev-middleware`, which put the output in memory.
So the `path` property doesn't have any impact.

Here, the documentation of `webpack-dev-middleware` explains :

```javascript
app.use(webpackMiddleware(webpack({
	// webpack options
	// webpackMiddleware takes a Compiler object as first parameter
	// which is returned by webpack(...) without callback.
	entry: "...",
	output: {
		path: "/"
		// no real path is required, just pass "/"
		// but it will work with other paths too.
	}
})
```